examples/basic_operations.js

```diff
@@ -1,27 +1,42 @@
 "use strict";
 
-var Redis = require("ioredis");
-var redis = new Redis();
+const Redis = require("ioredis");
+const redis = new Redis(process.env.redisPort, process.env.redisEndpoint, {password: process.env.redisPW});
 
 // ioredis supports all Redis commands:
-redis.set("foo", "bar");
+redis.set("foo", "bar"); // returns promise which resolves to string, "Ok"
```

Comment:
The response should be `"OK"` (in uppercase)

Reply:
done